### PR TITLE
Update PPP - Jonah

### PR DESCRIPTION
**Pull request by jonahtanjz**

### CODE REVIEW

File: docs/team/jonahtanjz.md [hunk 1]
```diff
@@ -43,6 +46,7 @@ This project is based on the AddressBook-Level3 project created by the [SE-EDU i
   * Added validation for user input
     * Check for invalid prefixes in command
     * Check for duplicate prefixes that are not supposed to be repeated in the command 
+  * Refactor AB3 code to adapt to SimplyKitchen
```
Comment (songyiang) [inline docs/team/jonahtanjz.md]: Will "Refactored" be better?

**Review by songyiang — APPROVED** [review]
LGTM except for a minor suggestion.

File: docs/team/jonahtanjz.md (hunk 1 above)
Comment (jonahtanjz) [reply docs/team/jonahtanjz.md]: Thanks

**Review by songyiang — APPROVED** [review]
LGTM

**Pull request merged**